Sources/Impl/PinLayoutImpl+Coordinates.swift

```diff
@@ -134,6 +147,13 @@ extension PinLayoutImpl {
         setBottom(layoutSuperviewRect.height - value, context)
         return self
     }
+
+    @discardableResult
+    internal func bottom(_ percent: Percent, _ context: Context) -> PinLayout {
+        guard let layoutSuperviewRect = layoutSuperviewRect(context) else { return self }
```

Comment:
Could you update the method `func bottom(_ percent: Percent) -> PinLayout` in PinLayoutImpl.swift to:
```
func bottom(_ percent: Percent) -> PinLayout {
        return bottom(percent, { return "bottom(\(percent.description))" })
    }
```